src/cockpit/389-console/src/dsModals.jsx

```diff
@@ -493,33 +400,14 @@ export class CreateInstanceModal extends React.Component {
             createTLSCert,
             createInitDB,
             createOK,
-            modalMsg,
             errObj,
         } = this.state;
-        let errMsgClass = "";
-        let errMsg = "";
-        let createSpinner = "";
-        if (loadingCreate) {
-            createSpinner = (
-                <Row>
-                    <div className="ds-margin-top-lg ds-modal-spinner">
-                        <Spinner size="lg" />
-                        Creating instance...
-                    </div>
-                </Row>
-            );
-        }
 
-        if (modalMsg == "") {
-            // No errors, but to keep the modal nice and stable during input
-            // field validation we need "invisible" text to keep the modal form
-            // from jumping up and down.
-            errMsgClass = "ds-clear-text";
-            errMsg = "no errors";
-        } else {
-            // We have error text to report
-            errMsgClass = "ds-modal-error";
-            errMsg = modalMsg;
+        let saveBtnName = "Create Instance";
```

Comment:
When I create an instance, I've got a browser crush with the next error:

```
CMD: setServerId: Test if instance is running  ==> dsctl -j localhost status
cockpit.js:1 {
    "desc": "'NoneType' object has no attribute 'get'"
}
```

But when I reload the page, the instance is installed (but not started)
And when I try to start it in the UI, it hangs...

When I try to start the instance through CLI, it waits for input. So it may be a part of the problem, as UI doesn't like interactive input usually (but you can provide the input if you want. I've used it for inputing credentials - https://github.com/389ds/389-ds-base/blob/master/src/cockpit/389-console/src/lib/monitor/replMonitor.jsx#L1071 )

Reply:
I don't get a browser crash but I hit the NSS issue with lib389.  The UI also does not use the interactive installer but it creates a INF file (dscreate from-file).   I think a change in lib389 got pulled in on the rebase.  I will investigate...